std_msgs/Header header

uint16 gps_week_number
float64 gps_time_of_week
float32 heading_deg
float32 heading_std_deg
float32 baseline_length
float32 pitch_deg
float32 pitch_std_deg